library/objective-c/EnvoyEngine.m

```diff
@@ -274,9 +282,9 @@ - (int)cancel {
     ios_on_cancel(context);
     // Step 3: propagate the reset into native code.
     reset_stream(_streamHandle);
-    return 0;
+    return ENVOY_SUCCESS;
```

Comment:
While this is technically correct, and ultimately what we want, these enum values don't currently match the signature of the method.

Reply:
Ah you're right, for some reason I thought they were pure integers not an enum. Reverting this change

Reply:
I do think we should change these to an enum in the near future, I can open a PR for it later

Reply:
(Changing this over is currently held up by https://github.com/lyft/envoy-mobile/issues/333)